pkgs/by-name/hy/hyprland-iio/package.nix

```
@@ -0,0 +1,49 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchpatch,
+  meson,
+  cmake,
+  pkg-config,
+  dbus,
+  ninja,
+}:
+
+stdenv.mkDerivation {
+  pname = "iio-hyprland";
+  version = "0-unstable-2023-09-27";
+
+  src = fetchFromGitHub {
+    owner = "JeanSchoeller";
+    repo = "iio-hyprland";
+    rev = "f31ee4109379ad7c3a82b1a0aef982769e482faf";
+    hash = "sha256-P+m2OIVS8QSQmeVYVIgt2A6Q/I3zZX3bK9UNLyQtNOg=";
+  };
+
+  patches = [
+    # fix rotation of touch and tablet not working
+    (fetchpatch {
+      name = "keyword.patch";
+      url = "https://github.com/JeanSchoeller/iio-hyprland/commit/f13242bbdf4a0d062683bf23b2431bb72e2e6ab9.diff";
+      hash = "sha256-lp5wmvL+F209FAdEdrmvPkYWi2ZzJq+2jrpGdFlolqQ=";
+    })
```

```suggestion
    (fetchpatch2 {
      name = "keyword.patch";
      url = "https://github.com/JeanSchoeller/iio-hyprland/pull/14/commits/f13242bbdf4a0d062683bf23b2431bb72e2e6ab9.diff";
      hash = "sha256-RRjyVTPJj3+7q1eqqQXnH6izCa4fLbYXV6oWJqHuQFs=";
    })
```

As is, it seemed like the commit was from the main upstream, not a fork. This should make it clearer that it comes from a PR.
And I believe `fetchpatch2` is currently preferred.

---

https://github.com/JeanSchoeller/iio-hyprland/pull/14 got merged a while ago, so let's move to the latest unstable commit.